### PR TITLE
Related visitors API for SDKs

### PR DESCRIPTION
**Pull request by TheUnderScorer**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -59,32 +59,32 @@ export default {
...
         {
-          from: 'schemas/fingerprint-server-api.yaml',
+          from: 'schemas/fingerprint-server-api-for-sdks.yaml',
           to: 'schemas/fingerprint-server-api.yaml',
           transform: (content) => transformSchema(content),
         },
         {
-          from: 'schemas/fingerprint-related-visitors-api.yaml',
+          from: 'schemas/fingerprint-related-visitors-api-readme-explorer.yaml',
           to: 'schemas/fingerprint-related-visitors-api.yaml',
```
Comment (JuroUhlar) [inline webpack.config.js:68]:
```suggestion
          to: 'schemas/fingerprint-related-visitors-api-readme-explorer.yaml',
```

File: webpack.config.js [hunk 2]
```diff
@@ -59,32 +59,32 @@ export default {
...
         },
         {
-          from: 'schemas/fingerprint-server-api.yaml',
+          from: 'schemas/fingerprint-server-api-readme-explorer.yaml',
           to: 'schemas/fingerprint-server-api-readme-explorer.yaml',
           transform: (content) => transformSchema(content, readmeApiExplorerTransformers),
         },
         {
-          from: 'schemas/fingerprint-server-api.yaml',
+          from: 'schemas/fingerprint-server-api-for-sdks.yaml',
```
Comment (JuroUhlar) [inline webpack.config.js:82]: Do we still need this? If `schemas/fingerprint-server-api.yaml` is now only used by SDKs maybe we can just apply `removeExtraDocumentation` there?

File: webpack.config.js [hunk 3]
```diff
@@ -59,32 +59,32 @@ export default {
...
           to: 'schemas/fingerprint-server-api-compact.yaml',
           transform: (content) => transformSchema(content, removeExtraDocumentationTransformers),
         },
         {
-          from: 'schemas/fingerprint-server-api.yaml',
+          from: 'schemas/fingerprint-server-api-for-sdks.yaml',
```
Comment (JuroUhlar) [inline webpack.config.js:87]: Do we still need this? If `schemas/fingerprint-server-api.yaml` is now only used by SDKs maybe we can just apply this transformer there?

File: webpack.config.js [hunk 4]
```diff
@@ -59,32 +59,32 @@ export default {
...
           to: 'schemas/fingerprint-server-api.yaml',
           transform: (content) => transformSchema(content),
         },
         {
-          from: 'schemas/fingerprint-related-visitors-api.yaml',
-          to: 'schemas/fingerprint-related-visitors-api.yaml',
+          from: 'schemas/fingerprint-related-visitors-api-readme-explorer.yaml',
+          to: 'schemas/fingerprint-related-visitors-api-readme-explorer.yaml',
```
Comment (ilfa) [inline webpack.config.js:68]: @JuroUhlar could we simply change the artefact name?

File: webpack.config.js (hunk 3 above)
Comment (TheUnderScorer) [reply webpack.config.js:77]: Fixed!

File: webpack.config.js (hunk 4 above)
Comment (JuroUhlar) [reply webpack.config.js:63]: What do you mean? remove transformers?

File: webpack.config.js [hunk 5]
```diff
@@ -59,32 +59,22 @@ export default {
     new CopyWebpackPlugin({
       patterns: [
         {
-          from: 'schemas/fingerprint-server-api.yaml',
-          to: 'schemas/fingerprint-server-api.yaml',
```
Comment (ilfa) [inline webpack.config.js:63]: I think we should keep this schema, we mention it in the documentation and customers could relay on it

Comment (TheUnderScorer) [reply webpack.config.js:63]: Fixed

**Review by ilfa — APPROVED** [review]

**Pull request merged**